themes/default/assets/config/tailwind.config.js

```diff
@@ -20,9 +20,20 @@ const brand = {
 
 const white = defaultTheme.colors.white;
 const black = defaultTheme.colors.black;
-const red = defaultTheme.colors.red;
 const transparent = defaultTheme.colors.transparent;
 
+const red = {
```

Comment:
Up to now, we've been using Tailwind's v1 defaults for `red`, which changed in v2. These are the actual v1 values. https://tailwindcss.com/docs/upgrading-to-v2#configure-your-color-palette-explicitly